android/src/main/java/com/capacitorjs/community/plugins/bluetoothle/Device.kt

```diff
@@ -10,12 +11,32 @@ import android.os.Handler
 import android.os.Looper
 import com.getcapacitor.Logger
 import java.util.*
+import java.util.concurrent.ConcurrentLinkedQueue
+import java.util.concurrent.atomic.AtomicReference
 
 class CallbackResponse(
     val success: Boolean,
     val value: String,
 )
 
+class WriteTimeout(
```

Comment:
Many calls have a timeout, not only write. Should we call this `TimeoutHandler`?